pyplanet/apps/contrib/brawl_match/__init__.py

```diff
@@ -74,9 +74,14 @@ def __init__(self, *args, **kwargs):
 	async def start_match_command(self, player, *args, **kwargs):
 		if self.match_tasks:
 			await self.brawl_chat(f'A match is currently in progress!', player)
+		elif not await self.check_maps_on_server():
+			await self.brawl_chat(f'Not all maps for the match are on the server', player)
 		else:
 			await self.register_match_task(self.start_match, player)
 
+	async def check_maps_on_server(self):
+		return all(map(lambda t: self.instance.map_manager.playlist_has_map(t[0]), self.brawl_maps))
```

Comment:
`return all([self.map_manager.playlist_has_map(map) for map in self.brawl_maps])`

Generally list comprehensions are better Python [solutions](https://stackoverflow.com/questions/1247486/list-comprehension-vs-map) compared to `map` and `filter` functions.